osf/models/mixins.py

```diff
@@ -691,10 +691,7 @@ def get_reviewable_state_counts(self):
     def get_request_state_counts(self):
         # import stuff here to get around circular imports
         from osf.models import PreprintRequest
-        qs = PreprintRequest.objects.filter(target__provider__id=self.id,
-                                            target__node__isnull=False,
-                                            target__node__is_deleted=False,
-                                            target__node__is_public=True)
+        qs = PreprintRequest.objects.filter(target__provider__id=self.id)
```

Comment:
I talked with Dawn about this more. It seems like this should really filter out requests tied to private and deleted preprints and the list filter:
https://github.com/CenterForOpenScience/osf.io/blob/b492411b7d1084dbc7e74758f85cc6eb85f40f32/api/providers/views.py#L410
should be updated to do the same.

This is because `private` is currently used to hide spammy preprints and `deleted` preprints should be excluded.